ui/src/main/resources/edu/wpi/grip/ui/codegeneration/CodegenDialog.fxml

```diff
@@ -24,23 +24,26 @@
             <rowConstraints>
                 <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="NEVER" />
                 <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="NEVER" />
+            <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="NEVER" />
```

Comment:
This isn't aligned correctly. Tabs vs spaces?

Reply:
Hmmm possibly.  I didn't edit that file manually so I guess SceneBuilder didn't align.  I'll update manually